package.nuspec

```diff
@@ -31,11 +31,14 @@
     <file src="DerivedData\Build\Products\Debug\FluentUI-React-Native-Button\libFluentUI-React-Native-Button.a" target="Debug-macosx"/>
     <file src="DerivedData\Build\Products\Debug\FluentUI-React-Native-FocusZone\libFluentUI-React-Native-FocusZone.a" target="Debug-macosx"/>
     <file src="DerivedData\Build\Products\Debug\FluentUI-React-Native-FocusZone\libFluentUI-React-Native-MenuButton.a" target="Debug-macosx"/>
```

Comment:
There is a typo I made here in both debug and ship. If you can fix it that would be great lol